kubernetes/charts/weblogic-operator/values.yaml

```diff
@@ -167,3 +167,21 @@ externalDebugHttpPort: 30999
 # such as "name, claimName, volumeName", or leave it commented out to use 
 # the default list of field names.
 # dns1123Fields: ""
+
+# introspectorJobNameSuffix overrides the default suffix that the operator uses
+# to append to the domainUID to form the name of the domain introspector job name.
+# Note that the resultant job name should not be more than 58 characters due to
+# the Kubernetes' limit to the name of a job and kubernetes appends additional 5
```

Comment:
the Kubernetes' limit -> the Kubernetes limit (no apostrophe needed)
kubernetes appends additional 5 -> Kubernetes appends five additional